### DISCUSSION TITLE
Is this supported? Top-Third, Middle-Third, Bottom-Third for Vertical Displays

### DISCUSSION THREAD

**Discussion by seanahrens** [rxhanson/RectanglePro-Community#165] [Q&A]
I have a vertically-oriented second display. The most common app layout I desired on that display is three apps, stacked vertically, (each with width 100% and height 33.3%). 

In the long list of pre-specified layout types in RectanglePro, I do not see layout options that support this. The available "thirds" are horizontal-thirds. I would like to specify drag-and-drop edges and corners which will drop the application into one of these three thirds.

![Screenshot 2022-11-09 at 3 33 01 PM](https://user-images.githubusercontent.com/109729/200935878-f8f26ca3-c231-4c2b-b832-4c8d5ea13e85.png)

Is there a way to achieve my goal with RectanglePro? Thank you.

**Comment by rxhanson** [comment]
The default selection for the sides of the portrait display is thirds. Is that what you're looking for?

**Reply by seanahrens** [reply]
Thanks. I see that now. That will work well enough to solve my current needs. 

I still notice, however, that vertical thirds options are absent from the drop down menus, meaning I am not able to assigns corners or the top or bottom trigger areas to place the window as a vertical third. Or assign hotkeys to "vertical thirds". Since I'm real new to the app, I'm guessing there may be other features that rely on that set of predefined layout type that I haven't yet discovered.

In any case, I can hobble by with the existing implementation, but wanted to check that I wasn't missing a way to add vertical thirds as members of that drop down? And that the app's support of vertical displays in a sense takes a backseat to horizontal? I get it that all apps have to make tradeoffs, I just want to make sure that there isn't a workflow I haven't discovered that makes it support these vertical thirds as well as it supports horizontal ones.

Thanks for your work in developing it regardless! So far it's showing itself quite useful already.

**Reply by rxhanson** [reply]
Certain window actions, like thirds, will automatically orient themselves on portrait displays, hence the nomenclature "First Third" rather than "Left Third". Let me know if for some reason something's not working right with these.

**Reply by rxhanson** [reply]
Forgot to mention that it's also on my todo list for an upcoming release to allow custom shortcuts to be selected in edge snap areas.

**Reply by seanahrens** [reply]
"Certain window actions, like thirds, will automatically orient themselves on portrait displays" Oh fascinating. I did not realize that. I used the iconography as a guide to how they would behave, ruling them out for my use case. Great to know. I'll play with those when I next fiddle with my set up.
